source/_docs/terminus/plugins/directory.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Terminus Plugin Directory
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+permalink: docs/terminus/plugins/:basename/
+---
+The following plugins are supported for Terminus 1.0:
+<div class="row">
+  <div style="margin-bottom:30px;" class="col-md-6">
+    <div class="topic-info">
+      <div class="pantheon-official">
+        <p>Pantheon Official</p>
```

Comment:
We should credit the original author, Brian Thompson (https://github.com/rvtraveller) for the Terminus Composer plugin, not me. Only link to the Pantheon-Systems repository, though. (There is a link to the original repository in the Pantheon-Systems repository.)